sidecar/sokuji_sidecar/gpt_sovits/g2p/english/Normalization.py

```
@@ -0,0 +1,289 @@
+
+    num = int(num_str)
+    if num == 0: return 'zero'
+
+    units = ["", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine"]
+    teens = ["ten", "eleven", "twelve", "thirteen", "fourteen", "fifteen", "sixteen", "seventeen", "eighteen",
+             "nineteen"]
+    tens = ["", "", "twenty", "thirty", "forty", "fifty", "sixty", "seventy", "eighty", "ninety"]
+    thousands = ["", "thousand", "million", "billion", "trillion"]
+
+    def convert_less_than_thousand(n):
+        if n == 0: return ""
```

Ruff (0.15.21)

[error] Multiple statements on one line (colon)

(E701)